qemu/tests/rh_kselftests_vm.py

```diff
@@ -0,0 +1,58 @@
+from avocado.core import exceptions
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    rh_kselftests_vm test
+    1) Download the current kernel selftests RPM
+    2) Install the RPM
+    3) Execute the kernel selftests
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+    kernel_path = params.get("kernel_path", "/tmp/kernel")
+    tests_execution_cmd = params.get("tests_execution_cmd")
+    whitelist = params.get("whitelist").split()
```

Comment:
Align with the comment above,  now, only s390x have the known skip case, so:
`whitelist = params.get("whitelist", "").split()`  or  `whitelist = params.objects("whitelist")`

Reply:
Done !